scripts/exgdas_atmos_verfrad.sh

```diff
@@ -173,17 +173,17 @@ if [[ -s ${radstat} && -s ${biascr} ]]; then
     ${USHradmon}/radmon_verf_bcoef.sh ${PDATE}
     rc_bcoef=$?
 
-    ${USHradmon}/radmon_verf_bcor.sh ${PDATE}
+    ${USHradmon}/radmon_verf_bcor.sh "${PDATE}"
     rc_bcor=$?
 
-    ${USHradmon}/radmon_verf_time.sh ${PDATE}
+    ${USHradmon}/radmon_verf_time.sh "${PDATE}"
     rc_time=$?
 
     #--------------------------------------
     #  optionally run clean_tankdir script
     #
     if [[ ${CLEAN_TANKVERF:-0} -eq 1 ]]; then
-       ${USHradmon}/clean_tankdir.sh glb 60
+       "${USHradmon}/clean_tankdir.sh glb 60"
```

Comment:
Only the first argument should be in quotations

Reply:
That makes sense, thanks.